lib/binding.ts

```diff
@@ -135,100 +135,100 @@ export interface PIDCoefficients {
+  read(numBytesToRead: number): Promise<number[]>;
+  write(bytes: number[]): Promise<void>;
 }
 
 export declare class RevHub {
   constructor();
-  open(serialPort: Serial, destAddress: number): Promise<{resultCode: number}>;
+  open(serialPort: Serial, destAddress: number): Promise<void>;
   isOpened(): boolean;
   close(): void;
   setDestAddress(destAddress: number): void;
   getDestAddress(): number;
   setResponseTimeoutMs(responseTimeoutMs: number): void;
   getResponseTimeoutMs(): number;
-  sendWriteCommandInternal(packetTypeID: number, payload: number[]): Promise<{resultCode: number}>;
-  sendWriteCommand(packetTypeID: number, payload: number[]): Promise<{value?: number[], resultCode: number}>;
-  sendReadCommandInternal(packetTypeID: number, payload: number[]): Promise<{resultCode: number}>;
-  sendReadCommand(packetTypeID: number, payload: number[]): Promise<{value?: number[], resultCode: number}>;
-  getModuleStatus(clearStatusAfterResponse: boolean): Promise<{value?: ModuleStatus, resultCode: number}>;
-  sendKeepAlive(): Promise<{resultCode: number}>;
-  sendFailSafe(): Promise<{resultCode: number}>;
-  setNewModuleAddress(newModuleAddress: number): Promise<{resultCode: number}>;
-  queryInterface(interfaceName: string): Promise<{value?: ModuleInterface, resultCode: number}>;
-  setModuleLEDColor(red: number, green: number, blue: number): Promise<{resultCode: number}>;
-  getModuleLEDColor(): Promise<{value?: RGB, resultCode: number}>;
-  setModuleLEDPattern(ledPattern: LEDPattern): Promise<{resultCode: number}>;
-  getModuleLEDPattern(): Promise<{value?: LEDPattern, resultCode: number}>;
-  setDebugLogLevel(debugGroup: DebugGroup, verbosityLevel: VerbosityLevel): Promise<{resultCode: number}>;
-  static discoverRevHubs(serialPort: Serial): Promise<{value?: DiscoveredAddresses, resultCode: number}>;
-  getInterfacePacketID(interfaceName: string, functionNumber: number): Promise<{value?: number, resultCode: number}>;
+  sendWriteCommandInternal(packetTypeID: number, payload: number[]): Promise<void>;
+  sendWriteCommand(packetTypeID: number, payload: number[]): Promise<number[]>;
+  sendReadCommandInternal(packetTypeID: number, payload: number[]): Promise<void>;
+  sendReadCommand(packetTypeID: number, payload: number[]): Promise<number[]>;
+  getModuleStatus(clearStatusAfterResponse: boolean): Promise<ModuleStatus>;
+  sendKeepAlive(): Promise<void>;
+  sendFailSafe(): Promise<void>;
+  setNewModuleAddress(newModuleAddress: number): Promise<void>;
+  queryInterface(interfaceName: string): Promise<ModuleInterface>;
+  setModuleLEDColor(red: number, green: number, blue: number): Promise<void>;
+  getModuleLEDColor(): Promise<RGB>;
+  setModuleLEDPattern(ledPattern: LEDPattern): Promise<void>;
+  getModuleLEDPattern(): Promise<LEDPattern>;
+  setDebugLogLevel(debugGroup: DebugGroup, verbosityLevel: VerbosityLevel): Promise<void>;
+  static discoverRevHubs(serialPort: Serial): Promise<DiscoveredAddresses>;
```

Comment:
Since this is `static` I think it would be clearer to list it at the top, even above the constructor.

Reply:
Disregard this for now.